solutions/opensearch-cross-cluster-reoplication/terraform/provider.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_providers {
+    aiven = {
+      source = "aiven/aiven"
+      version = ">= 2.5.0, < 3.0.0" }
```

Comment:
3.11.0 is out, perhaps test it against the new version to make sure it works and remove the constraint `3.0.0`?
https://github.com/aiven/terraform-provider-aiven/releases/tag/v3.11.0

Reply:
@runwuf - Updated and tested, can u pls check now? thnx